README.md

```diff
@@ -77,8 +77,11 @@ containerd offers a full client package to help you integrate containerd into yo
 ```go
 
 import (
+  "context"
+  
```

Comment:
Seems Git is not happy about the whitespace here.

Reply:
I didn't know `git show --check` is [checking for whitespace](https://git-scm.com/docs/git-show#Documentation/git-show.txt---check) errors. That's cool. Should be fixed at 0b78690.